ministryofjustice/correspondence_tool_staff
# CT-2902 Prevent manager users from creating new Offender SAR

## Description
Manager users could go directly to https://development.track-a-query.service.justice.gov.uk/cases/offender_sars/new and create offender SARs, though they couldn't view them afterwards. Fixed and amended tests that weren't using correct user.

## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/1161161/85123416-25237600-b220-11ea-8f94-08e8a96cb9bc.png)

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2902

### Deployment
nope
### Manual testing instructions
Log in as David Attenborough or any manager user and go to /cases/offender_sars/new - it should redirect back to the home page with a warning message.
